### PR TITLE
Landing page fixes

### PR DESCRIPTION
**Pull request by johndcollett**
### Description

fixes up some sections from the landing pages and incorporates the changes made in this PR: https://github.com/Shopify/hydrogen-react/pull/162

### Additional context

rendered content in shopify-dev: https://shopify-dev.raect-hydrogen.john-collett.us.spin.dev/api/hydrogen-react

### CODE REVIEW

File: packages/react/docs/generated/generated_static_pages.json [hunk 1]
```diff
@@ -8,49 +8,13 @@
         "type": "Generic",
         "anchorLink": "setup",
         "title": "Setup",
-        "sectionContent": "\n1. Run one of the example commands to install the package\n1. Import the component, hook, or utility that you want to use in your app. Refer to the list of [what's available](components) in Hydrogen React.\n      ",
+        "sectionContent": "\n1. Run one of the example commands to install the package\n1. Import the component, hook, or utility that you want to use in your app. For more detailed instructions, see the Getting Started Guide.\n      ",
```
Comment (frehner) [inline packages/react/docs/generated/generated_static_pages.json:11]: Can we link to the guide?

File: packages/react/docs/generated/generated_static_pages.json [hunk 2]
```diff
@@ -69,60 +33,64 @@
...
-            },
+            }
+          ]
+        }
+      },
+      {
+        "type": "Generic",
+        "anchorLink": "hooks",
+        "title": "Hooks",
+        "sectionContent": "Hooks are functions that provide reusable, business and/or stateful logic. These functions allow you to leverage the business logic and state management functions used in the components with more flexibility.",
```
Comment (frehner) [inline packages/react/docs/generated/generated_static_pages.json:74]:
```suggestion
        "sectionContent": "Hooks are functions that provide reusable, business, and/or stateful logic. These functions allow you to leverage the business logic and state management functions used in the components with more flexibility.",
```

Thoughts on the oxford comma? haha

File: packages/react/docs/generated/generated_static_pages.json [hunk 3]
```diff
@@ -137,20 +105,20 @@
         "title": "Resources",
         "resources": [
           {
-            "name": "Hydrogen framework",
-            "subtitle": "Hydrogen React includes a framework that offers a set of best practices and scaffolding for building a website.",
-            "url": "/api/hydrogen",
+            "name": "Custom Storefronts",
+            "subtitle": "Learn more about how to design, build, and manage, your custom storefront.",
```
Comment (frehner) [inline packages/react/docs/generated/generated_static_pages.json:109]:
```suggestion
            "subtitle": "Learn more about how to design, build, and manage your custom storefront.",
```

**Review by frehner — COMMENTED** [review]
Thanks!

File: packages/react/docs/generated/generated_static_pages.json (hunk 2 above)
Comment (abharvey) [reply packages/react/docs/generated/generated_static_pages.json:74]: Always Oxford 🎩

File: packages/react/docs/generated/generated_static_pages.json (hunk 1 above)
Comment (abharvey) [reply packages/react/docs/generated/generated_static_pages.json:11]: Already is good sir :)

image

**Comment by frehner** [comment]
I think Helen just merged in some changes, so you need to pull in from the base and generate the file again.

**Review by frehner — APPROVED** [review]

**Pull request merged**